### ISSUE TITLE
lint: flag provided but not defined: -skip-deps

### ISSUE THREAD

**Issue by Morriz** [roboll/helmfile#581]
Why is `--skip-deps` not available for linting? I'd like to do the following:

```
helmfile repos
helmfile -e dev lint --skip-deps
helmfile -e tst lint --skip-deps
helmfile -e acc lint --skip-deps
helmfile -e prd lint --skip-deps
```

**Comment by mumoshu** [comment]
@mumoshu Good catch! We should add `--skip-deps` to `lint` as well.

Would you mind submitting a PR for this?

Perhaps in `main.go` you'd add:

```
cli.BoolFlag{
	Name:  "skip-deps",
	Usage: "skip running `helm repo update` and `helm dependency build`",
},
```

to the args of `lint`, and wrap `SyncReponsOnce` which is the core of the `repos` command, like this:

```
if !c.Bool("skip-deps") {
	if errs := ctx.SyncReposOnce(state, helm); errs != nil && len(errs) > 0 {
		return errs
	}
	if errs := state.BuildDeps(helm); errs != nil && len(errs) > 0 {
		return errs
	}
}
```

**Comment by Morriz** [comment]
eh...I have not really set up a golang toolchain. I would have to start programming in go first to feel comfortable doing PRs...did you not just now give the solution already?

**Comment by mumoshu** [comment]
@Morriz Don't worry! It's just my habit to invite everyone to contribute :) I've submitted #602 for this.

**Comment by mumoshu** [comment]
Closing as resolved via #602
